frontend/src/components/worksheets/items/MarkdownItem.js

```diff
@@ -191,6 +196,13 @@ class MarkdownItem extends React.Component {
             //   start = 0, inStart = 2, inEnd = 5, end = 7
             var start = text.indexOf('$', curr);
             if (start === -1) break; // No more math blocks
+            if (start > 0 && text[start - 1] === '\\') {
+                // \$ --> \\$
```

Comment:
I'm a bit confused ... when would we find `\$` in the markdown cell text? I would have expected to only see `$`.

Reply:
When a user types a text like `$ THIS IS PLAIN TEXT $`, the text will be treated as a latex formula, and no `$` will be displayed in the markdown cell. So if a user wants to display `$`, he should use type `\$` instead.